settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
This is my favorite file. How deep and insightful it is.

Reply:
:D